packages/hadron-document/index.d.ts

```diff
@@ -112,10 +112,12 @@ declare const HadronDocumentEvents: {
 
 declare class HadronDocument extends EventEmitter {
   constructor(doc: unknown, cloned: boolean);
+  uuid: string;
   type: 'Document';
   currentType: 'Document';
   elements: LinkedList<HadronElement>;
   cancel(): void;
+  apply(doc: unknown): void;
```

Comment:
Maybe `insertBeginning()` should be added here as well?

Reply:
Totally, thanks! Added in https://github.com/mongodb-js/compass/pull/2929/commits/8586328d38076aaa5f6b9508d3395c6a507bb582